geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -2473,7 +2475,7 @@
       "The IP address on which the Server will be bound.  By default, the Server is bound to all local addresses.";
   public static final String START_SERVER__CACHE_XML_FILE = CACHE_XML_FILE;
   public static final String START_SERVER__CACHE_XML_FILE__HELP =
-      "Specifies the name of the XML file or resource to initialize the cache with when it is created.";
+      "Specifies the name of the XML file or resource to initialize the cache with when it is created. NOTE: If cluster configuration is enabled, then it will take precedence over the XML file provided in this field";
```

Comment:
Instead of "XML file provided in this field" I would say "cache-xml-file". or "cache XML file"